server/rollback.go

```diff
@@ -27,9 +26,8 @@ application.
 `,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := GetServerContextFromCmd(cmd)
-			cfg := ctx.Config
-			home := cfg.RootDir
-			db, err := openDB(home, GetAppDBBackend(ctx.Viper))
+
+			db, err := openDB(ctx.Config.RootDir, GetAppDBBackend(ctx.Viper))
```

Comment:
Should we have a `defer db.Close()` right after the error check down below?

Reply:
I think this might need to be done in another commit as there is a lot of logic like this that is not handled either.